csharp/test/Microsoft.ML.OnnxRuntime.Tests.Android/RunAllTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using OpenQA.Selenium.Appium;
+using OpenQA.Selenium;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ML.OnnxRuntime.Tests.Android
```

Comment:
Needs comments here connecting this up to the DeviceRunner implementation. Links to the files in that which define the UI and have the elements we're looking for would be great as we'll need to check there if this breaks due to an update in that package.